### PR TITLE
The child object and functionality

### PR DESCRIPTION
**Pull request by sheldonhunterst**

### CODE REVIEW

File: Assets/Scripts/ChildObject.cs [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+        }
+        if (this.doesRepsoitionAirwayWork)
+        {
+            this.doesRescueBreathsWork = false;//this should be what forces a refactor of the breathing rate
+        }
+        else
+        {
+            this.doesRescueBreathsWork = true;//this should be what forces a refactor of the breathing rate
+        }
+        this.canRespond = RandomTrueOrFalse();
```
Comment (arimunoz) [inline Assets/Scripts/ChildObject.cs]: wondering why this.canRespond = RandonTrueOrFalse(); is here and not after in line 78. This code is repeated in line 96. Was it meant to be there?

File: Assets/Scripts/ChildObject.cs [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+                value = -1;
+                break;
+        }
+
+        return value;
+    }
+
+    public void RepositionedTheAirway()
+    {
+        if (this.doesRepsoitionAirwayWork)
```
Comment (arimunoz) [inline Assets/Scripts/ChildObject.cs]: Once you change the spelling in line 13; change this name. Also it's a method right? So, it should look like this.doesRespositionAirwayWork():

File: Assets/Scripts/ChildObject.cs [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+        switch (randomNumber)
+        {
+            case 0://8
+                value = 8;
+                break;
+            case 1://12
+                value = 12;
+                break;
+            case 2://52
+                value = 52;
```
Comment (arimunoz) [inline Assets/Scripts/ChildObject.cs]: test

File: Assets/Scripts/ChildObject.cs [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+    {
+        return this.canUnderstand;
+    }
+    public bool IsRespondingCoherently()
+    {
+        return this.canRespond;
+    }
+
+    // Use this for initialization
+    void Start () {
```
Comment (arimunoz) [inline Assets/Scripts/ChildObject.cs]: Do you plan on writing something in here or are you not sure what to put in here?

Comment (sheldonhunterst) [reply Assets/Scripts/ChildObject.cs]: Currently, that was auto-generated, so I don't know if I need it or if I need something else for that functionality.

Comment (arimunoz) [reply Assets/Scripts/ChildObject.cs]: ok

**Pull request merged**